modules/installation-configuration-parameters.adoc

```diff
@@ -1032,6 +1032,72 @@ If you deploy to a custom subnet, you cannot specify an external DNS server to t
+features, enabling customers to test functionality and provide feedback during
+the development process.
+
+For more information about the support scope of Red Hat Technology Preview features, see link:https://access.redhat.com/support/offerings/techpreview/[Technology Preview Features Support Scope].
+====
+// Undefine {FeatureName} attribute, so that any mistakes are easily spotted
+:!FeatureName:
+
+{rh-openstack-first} deployments do not have a single implementation of failure domains. Instead, availability zones are defined individually for each service, such as the compute service, Nova; the networking service, Neutron; and the storage service, Cinder. 
+
+Beginning with {product-title} 4.13, there is a unified definition of failure domains for {rh-openstack} deployments that covers all supported availability zone types. You can use failure domains to control related aspects of Nova, Neutron, and Cinder configurations from a single place. 
+
+In {rh-openstack}, a port describes a network connection and maps to an interface inside a compute machine. A port also:
+* Is defined by a network or by one more or subnets
+* Connects a machine to one or more subnets
+
+Failure domains group the services of your deployment by using ports. If you use failure domains, each machine connects to:
+* The `portTarget` object with the ID `control-plane`.
+* All non-control-plane `portTarget` objects within its own failure domain.
+* All networks in the machine pool's `additionalNetworkIDs` list.
+
+To configure failure domains for a machine pool, edit availability zone and port target parameters under `controlPlane.platform.openstack.failureDomains`.
+
+.{rh-openstack} parameters for failure domains
+[cols=".^2,.^3a,.^3a",options="header"]
+|====
+|Parameter|Description|Values
+
+|`platform.openstack.failuredomains.computeAvailabilityZone`
+|An availability zone for the server. If not specified, the cluster default is used.
+|The name of the availability zone. For example, `nova-1`.
+
+|`platform.openstack.failuredomains.storageAvailabilityZone`
+|An availability zone for the root volume. If not specified, the cluster default is used.
+|The name of the availability zone. For example, `cinder-1`.
+
+|`platform.openstack.failuredomains.portTargets`
+|A list of `portTarget` objects, each of which contextualizes a port target within a single failure domain. Each object must include an `id` parameter that specifies a port target.
+|A list of `portTarget` objects.
+
+|`platform.openstack.failuredomains.portTargets.portTarget.id`
+|The ID of an individual port target. Set the value of this parameter to `control-plane` to select that port target as the first network for machines. If this parameter is not set, the cluster default network is used.
+|`control-plane` or an arbitrary string.
+
+|`platform.openstack.failuredomains.portTargets.portTarget.network`
+|Required. The name or ID of a network to use in place of the primary cluster network.
```

Comment:
More like:

> The name or ID of the network to attach to machines in the failure domain.

It is only the primary cluster network IF the `id` is set to `control-plane`.

Reply:
```suggestion
|Required. The name or ID of the network to attach to machines in the failure domain.
```